src/commands/Management/Starboard/starboard.js

```diff
@@ -23,24 +23,22 @@ module.exports = class extends Command {
         const fetch = await starChannel.messages.fetch({ limit: 100 });
         const starMsg = fetch.find(m => m.embeds[0] && m.embeds[0].footer && m.embeds[0].footer.text.startsWith("⭐") && m.embeds[0].footer.text.endsWith(Message.id));
 
-        const jumpString = `[► View The Original Message](https://discordapp.com/channels/${Message.guild.id}/${Message.channel.id}/${Message.id})\n`;
```

Comment:
Still need this so it can say "View Original Message".

Reply:
Thats what it does.